control-plane/p4RuntimeArchStandard.h

```diff
@@ -786,6 +790,57 @@ class P4RuntimeArchHandlerCommon : public P4RuntimeArchHandlerIface {
             }
         }
 
+        // By default, an action profile uses the SumOfWeights semantics.
+        auto selectorSizeSemanticsAnnotation =
+            actionProfile.annotations->getAnnotation("selector_size_semantics");
+        if (selectorSizeSemanticsAnnotation) {
+            if (actionProfile.type == ActionProfileType::INDIRECT_WITH_SELECTOR) {
+                auto selectorSizeSemantics = selectorSizeSemanticsAnnotation->expr[0]->toString();
+                CHECK_NULL(selectorSizeSemantics);
+                // The expression may only contain 'sum_of_weights' or 'sum_of_members'
```

Comment:
Nit: This is actually "sum_of_weights" and "sum_of_members"? Same applies to the error message a couple lines down.

Reply:
Ah, good point. It's a little confusing for users though, because if they actually put `@selector_size_semantics(sum_of_weights)` P4 would get confused and think it was an identifier (at least it does if I try it). So they need to put `@selector_size_semantics("sum_of_weights")`. But if I say the expression needs to be `\"sum_of_weights\"`, then I worry they'll put `@selector_size_semantics("\"sum_of_weights\"")`... Wdyt?

Reply:
At least for the error message `\"sum_of_weights\"` should turn into `"sum_of_weights"` no?

Ha we can leave it as is. It looks like no matter what we do here there will be a user that could be confused.

Reply:
Yeah :'(. Fixed the error message though. I think that's a good point.